roles/basic_setup_rhel7/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "Subscription Manifest"
+  include_role:
+    name: theforeman.foreman.manifest
+
+- name: "Enable rhel-7-server-rpms"
+  include_role:
+    name: theforeman.foreman.repositories
+  vars:
+    products:
+      - name: Red Hat Enterprise Linux Server
+        repository_sets:
+          - name: Red Hat Enterprise Linux 7 Server (RPMs)
+            basearch: x86_64
+            releasever: 7Server
+
+- name: "Create Sync Plan"
+  include_role:
+    name: theforeman.foreman.sync_plans
+  vars:
+    sync_plans:
+      - name: "{{ sync_plan_name | default('RHEL Sync Plan') }}"
+        interval: "{{ sync_plan_interval | default(daily) }}"
+        cron_expression: "{{ sync_plan_cron_expression | default(omit) }}"
+        sync_date: "{{ sync_plan_sync_date | default('2020-01-01 00:00:00 UTC') }}"
+        products:
+          - Red Hat Enterprise Linux Server
+
+- name: "Create Activation Key"
+  include_role:
+    name: theforeman.foreman.activation_keys
+  vars:
+    activation_keys:
+      - name: "{{ activation_key_name }}"
```

Comment:
I think this should have a default and reduce the amount that a user needs to specify when using this role.

Reply:
I agree, although I'm struggling to come up with a good name. I'll go with something like `base_activation_key` for now since it's a default org view / library key with auto-attach and no subs.... which is about as basic as you can get for an activation key.